modules/ossm-rn-fixed-issues.adoc

```diff
@@ -21,6 +21,18 @@ The following issues been resolved in the current release:
 
 * https://issues.redhat.com/browse/OSSM-3025[OSSM-3025] Istiod sometimes fails to become ready. Sometimes, when a mesh contained many member namespaces, the Istiod pod did not become ready due to a deadlock within Istiod. The deadlock is now resolved and the pod now starts as expected.
 
```

Comment:
You should add the backticks to nodeSelector and tolerations

* https://issues.redhat.com/browse/OSSM-2493[OSSM-2493] Default `nodeSelector` and `tolerations` in SMCP not passed to Kiali. The nodeSelector and tolerations you add to `SMCP.spec.runtime.defaults` are now passed to the Kiali resource.